.travis.yml

```diff
@@ -109,6 +109,7 @@ script:
 after_script:
   - echo "after_script start"
   - source activate pandas-dev && pushd /tmp && python -c "import pandas; pandas.show_versions();" && popd
+  - python -c "from dateutil import tz; import datetime; print('Local TZ =', datetime.datetime.now(tz.tzlocal()).tzname())"
```

Comment:
`date +%Z`?

Reply:
why are you doing this here?

Reply:
i found it useful in debugging this issue, to spit out the timezone in the build environment (for travis its UTC). I could either remove this, or move it into one of the shell scripts (probably `ci/run_tests.sh`)

Reply:
i changed it to print out the local time, tz and utc offset. Let me know if you want it removed or moved to a script

Reply:
I think this may be useful (for the azure and travis build), but not sure if this is the right place for it.

Also since these tests are running on Linux, the `date` command should be sufficient.

Reply:
its ok to print it but do it in the install_travis script (and corresponding on azure). see where / how we print locale out.